crates/ln-dlc-node/Cargo.toml

```diff
@@ -42,16 +42,28 @@ serde = "1.0.147"
 serde_with = "3.1.0"
 sha2 = "0.10"
 time = "0.3"
-tokio = { version = "1", default-features = false, features = ["io-util", "macros", "rt", "rt-multi-thread", "sync", "net", "time", "tracing"] }
+tokio = { version = "1", default-features = false, features = ["io-util", "macros", "rt", "rt-multi-thread", "sync", "time", "tracing"] }
 tracing = "0.1.37"
 tracing-log = "0.1.3"
 ureq = "2.5.0"
 uuid = { version = "1.3.0", features = ["v4", "serde"] }
 
+axum = { version = "0.6", features = ["ws"], optional = true }
+tokio-tungstenite-wasm = { version = "0.3.0", features = ["native-tls"], optional = true }
+
+# To enable JS support when compiling under wasm
+[dependencies.getrandom]
```

Comment:
For some reason GitHub marks this in red 🤷

Reply:
Not sure why... apparently google says that it could be because of a syntax error somewhere, but it does seem to be correct

Reply:
I would just ignore this personally. It works and also validates using an online toml linter (https://www.toml-lint.com/)